core/src/main/java/com/google/errorprone/bugpatterns/nullness/NullnessFixes.java

```diff
@@ -44,7 +45,7 @@ private static String getQualifiedName(VisitorState state, SuggestedFix.Builder
     Symbol sym = FindIdentifiers.findIdent("Nullable", state, KindSelector.VAL_TYP);
     String defaultType =
         state.isAndroidCompatible()
-            ? "android.support.annotation.Nullable"
+            ? (state.isAndroidXCompatible() ? "androidx.annotation.Nullable" : "android.support.annotation.Nullable" )
```

Comment:
I'm not sure this is an intended use of these types of flags. The ErrorProne team would be able to state authoritatively, but perhaps switching this to a check for if the type is available in the VisitorState would be more appropriate?

@awturner

Reply:
An alternative would just be to accept a "nullableAnnotation" option to the build itself, similar to the way the android option works.

Reply:
I like the idea of passing in a flag with the fully qualified name of the nullableAnnotation to use.  I would not do it the way we do for -XDandroidCompatible though, I would use the Error Prong flags functionality: https://errorprone.info/docs/flags#pass-additional-info-to-bugcheckers

How about a flag like `-XepOpt:Nullness:NullableAnnotation`?

Reply:
That sounds good to me! I'll rework this PR to use that approach